src/workspaces/NewWorkspaceModal/NewWorkspaceModal.test.ts

```diff
@@ -445,8 +445,7 @@ describe('NewWorkspaceModal', () => {
   });
 
   describe('decides when to show a policy section ', () => {
-    const policyTitle = 'Security, controls, and workspace information';
-    const policyLabel = 'Workspace will inherit data protection policies';
+    const policyTitle = 'Security and controls on this workspace:';
     it('Shows a policy section when cloning an Azure workspace with polices', async () => {
```

Comment:
Unrelated nit: spelling

```suggestion
    it('Shows a policy section when cloning an Azure workspace with policies', async () => {
```

Reply:
Done